dlc-manager/src/payout_curve.rs

```diff
@@ -311,8 +311,8 @@ impl Evaluable for PolynomialPayoutCurvePiece {
             return if left_point.outcome_payout == right_point.outcome_payout {
                 right_point.outcome_payout as f64
             } else {
-                let slope = (right_point.outcome_payout - left_point.outcome_payout) as f64
-                    / (right_point.event_outcome - left_point.event_outcome) as f64;
+                let slope = (right_point.outcome_payout as f64 - left_point.outcome_payout as f64)
+                    / (right_point.event_outcome as f64 - left_point.event_outcome as f64);
```

Comment:
nit: as we expect event outcomes to be increasing only you don't really need to change this line

Reply:
Thanks for pointing this out. I've removed these changes.